charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```diff
@@ -21,15 +23,6 @@ spec:
             - --v=5
             - "--csi-address=unix://C:\\csi\\csi.sock"
             - --kubelet-registration-path={{ .Values.windows.kubeletRootDir }}\plugins\csi-secrets-store\csi.sock
-          lifecycle:
-              preStop:
```

Comment:
Just checking, are removing these hooks intentional?

Reply:
@tam7t that's correct. The hooks were removed as part of https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/366. 

The latest `node-driver-registrar` images are distroless so these prestop commands don't work. The `node-driver-registrar` cleans up the registration socket as part of container shutdown.

Reply:
Thanks SGTM